keyboards/reviung39/rules.mk

```diff
@@ -0,0 +1,82 @@
+# MCU name
+#MCU = at90usb1286
+MCU = atmega32u4
+
+# Processor frequency.
```

Comment:
Lines 5 to 39 can be removed, QMK provides these same values as defaults.

Reply:
thanks